src/Integration/RequestFetcher.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Psr\Http\Message\ServerRequestInterface;
+
+class RequestFetcher implements RequestFetcherInterface
```

Comment:
Please mark this class as `final` and add a small DocBlock to explain what it does